### ISSUE TITLE
[HW] Elisa Crescentini

### ISSUE THREAD

**Issue by AmyLynn87** [datastaxdevs/workshop-graphql-netflix#290]
**Name:** Elisa Crescentini
**Email:**  e.crescentini87@gmail.com
**Linkedin Profile:** https://www.linkedin.com/in/elisa-crescentini/

Attach the homework screenshots below:
-----------------------------------------

I finally finished these homework, after followed the workshop & the Ania Kubow Tutorial, I had some problem with publishing in Netlify until today, but I don't really understand what kind of screenshots do you need.
So...
This is the netlify link:  https://dreamy-mestorf-771edf.netlify.app/
and this is the github repository: https://github.com/AmyLynn87/netflix-clone-project

hope is ok and I will follow for sure the 3rd week in the summer series to watch again the netflix clone! maybe I'll understand more :)

![image](https://user-images.githubusercontent.com/82437140/125157194-c6ea6d80-e169-11eb-8d94-47f5e9367def.png)

**Comment by SonicDMG** [comment]
The screen you have above is fine. Were you able to add a custom video to your Netlifx app though? I took a look at the link you provided, that works fine and all, but I don't see a custom video. Great job on getting it deployed to Netlify BTW.

How did the Ania tutorial go for you?

**Comment by SonicDMG** [comment]
Even before we do the 3rd week of this series feel free to come reach out on Discord, LinkedIn, wherever if you have any questions or need any help.

**Comment by AmyLynn87** [comment]
I followed the workshop online with Ryan and Cedric but it was hard for me
to understand all the passages.
After that I followed Ania's tutorial. She is great, but I had some issues
with the installation of Netlify Cli, I fought with Powershell ( I have a
laptop and a desktop both with the latest Windows OS version, but I never
used the Powershell panel, so it takes a while for me to understand
everything, LOL.
And after all this adventure, I forgot to put the custom video!!
But.... what do you mean for custom video? I thought that you mean to
simply put a new thumbnail with name of the film, duration, synopsys... I'm
right?

Thanks


*Elisa Crescentini*
www.elisacrescentini.dev


Il giorno lun 12 lug 2021 alle ore 19:04 David Jones-Gilardi <
***@***.***> ha scritto:

**Comment by AmyLynn87** [comment]
I tried with gitpod during the workshop and I tried with my vscode with the
tutorial. I'm glad to did it because I learnt a lot, also if it was
frustrating, LOL.
So, I will try to put a custom video and I'll resend you the screenshot...
or I will wait the 3rd week :P
*Elisa Crescentini*
www.elisacrescentini.dev


Il giorno mar 13 lug 2021 alle ore 15:14 David Jones-Gilardi <
***@***.***> ha scritto:

**Comment by SonicDMG** [comment]
Whether you do that before or after the 3rd week is entirely up to you. I'll keep an eye on this issue regardless. Was the gitpod part frustrating or did that work ok for you?

**Comment by AmyLynn87** [comment]
The gitpod part was ok. The issues was with the GraphQL - first time that I saw a NoSQL DB!

Inviato da iPhone

**Comment by SonicDMG** [comment]
Ah, I see. Yea, I could see that being a little confusing if you had never done that before. Let's chat about it on Friday.

**Comment by SonicDMG** [comment]
Ok, per our convo in Discord all good on this.

**Comment by HadesArchitect** [comment]
Hey **Elisa Crescentini**, great job! Congrats! Here is your badge! https://api.badgr.io/public/assertions/rbYIoHsGSrOafHg90gdCXg
